src/components/intern-app.html

```diff
@@ -3,6 +3,10 @@
 <link rel="import" href="./user-list.html">
 <link rel="import" href="./edit-user.html">
 <link rel="import" href="./create-user.html">
+<!-- The core Firebase JS SDK is always required and must be listed first -->
+<script src="https://www.gstatic.com/firebasejs/6.4.0/firebase-app.js"></script>
+<!-- Firebase Database Link -->
+<script src="https://www.gstatic.com/firebasejs/5.0.1/firebase-database.js"></script>
```

Comment:
you should try adding these your your `package.json` over in-line scripting. I assume it's possible, though I don't fully know for sure. I think Christa was able to.

Reply:
Turned out I didn't need that at all. Removed it.